bin/reth/src/node/mod.rs

```diff
@@ -26,9 +26,8 @@ use reth_network::{
 use reth_primitives::{Account, Header, H256};
 use reth_provider::{db_provider::ProviderImpl, BlockProvider, HeaderProvider};
 use reth_stages::{
+    stage_metrics::HeaderMetrics,
```

Comment:
a question about the naming convention for the metrics modules, what do you think about following the structure
```
<crate>::metrics;
```
instead of, like in this case,
```
<crate>::<crate>_metrics;
```
I think it would be slightly nicer and less verbose